lib/msf/core/post/hardware/automotive/uds.rb

```diff
@@ -6,6 +6,22 @@ module Automotive
 
 module UDS
 
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Compat' => {
+          'Meterpreter' => {
+            'Commands' => %w[
+              automotive_active_bus
+              automotive_send_isotp_and_wait_for_response
+            ]
+          }
+        }
+      )
+    )
+  end
```

Comment:
After a conversation with @zeroSteiner. I'm going to be reverting any `/hardware/...` modules that were altered 
We don't believe the `/hardware/...` modules need the Meterpreter compatibility because they don't use Meterpreter sessions.